src/easyscience/variable/descriptor_number.py

```diff
@@ -50,6 +51,7 @@ def __init__(
         url: Optional[str] = None,
         display_name: Optional[str] = None,
         parent: Optional[Any] = None,
+        **kwargs: Any  # Additional keyword arguments (used for (de)serialization)
```

Comment:
what's the reason to move `_dependency_id` logic to `descriptor_number`? It is only `Parameter`s which will use it. No need to pollute the base class.